packages/wallet-ui/src/slices/walletSlice.ts

```diff
@@ -22,7 +23,10 @@ const initialState: WalletState = {
   isLoading: false,
   forceReconnect: false,
   accounts: [],
-  currentAccount: undefined,
+  currentAccount: {
+    address: DUMMY_ADDRESS,
+    addressIndex: 0,
+  } as Account,
```

Comment:
you can have a default account, so that you can reuse, same comment as the line on clearAccounts
```
const defaultAccount = {
    address: DUMMY_ADDRESS,
    addressIndex: 0,
} as Account
```

```suggestion
  currentAccount: defaultAccount,
```